src/linear_algebra/dimension.lean

```diff
@@ -9,167 +9,292 @@ import linear_algebra.basis
 import set_theory.ordinal
 noncomputable theory
 
-local attribute [instance, priority 0] classical.prop_decidable
+-- TODO : move
+@[simp] lemma nonempty_pempty : ¬ nonempty pempty :=
+assume ⟨h⟩, h.elim
```

Comment:
This can go in `logic.basic` in the misc section

Reply:
sorry, forgot to clean up this file.